API/RATMain.m

```diff
@@ -1,4 +1,21 @@
 function [problemStruct,results,bayesResults] = RATMain(problemStruct,controls)
+  % Select and run the internal RAT procedure.
+  %
+  % Parameters
+  % ----------
+  % problemStruct : struct
+  %     The project class in struct form (generated from parseClassToStructs)
+  % controls : struct
+  %     The controls struct (generated from parseClassToStructs)
```

Comment:
This routine will be called by both the matlab and python APIs. Therefore, the docstrings should apply to both, hence the  comment "(generated from parseClassToStructs)" should be changed.
Maybe "The input project cast to a struct" and similar for controls?